tools/ddsrouter_tool/src/cpp/user_interface/arguments_configuration.cpp

```diff
@@ -86,10 +87,25 @@ const option::Descriptor usage[] = {
         "  -d \t--debug\t  \t" \
         "Activate debug Logs (be aware that some logs may require specific CMAKE compilation options)." \
     },
+    {
+        optionIndex::VERSION,
+        0,
+        "v",
+        "version",
+        Arg::None,
+        "  -v \t--version\t  \t" \
+        "Print version, branch and commit hash." \
+    },
 
     { 0, 0, 0, 0, 0, 0 }
 };
 
+static void print_version()
+{
+    std::cout << "DDSRouter v" << DDSROUTER_CORE_VERSION_STR << "\ncommit hash: " << DDSROUTER_CORE_COMMIT_HASH <<
```

Comment:
NIT: we have discussed it and we think `v` must be inside `DDSROUTER_CORE_VERSION_STR`.
It requires a minor change in CMake file. Could you change that, please?